plutus-tx/src/PlutusTx/Lift.hs

```diff
@@ -44,6 +44,8 @@ import           Data.Proxy
 import           Data.Text.Prettyprint.Doc
 import qualified Data.Typeable                   as GHC
 
+type PrettyPrintable uni fun = ( PLC.GShow uni, PLC.Closed uni, uni `PLC.Everywhere` PrettyConst, Pretty fun)
```

Comment:
@effectfully I was thinking about adding such constraint somewhere, do you thing it is sufficient?

Reply:
Lo: https://github.com/input-output-hk/plutus/blob/master/plutus-core/untyped-plutus-core/src/UntypedPlutusCore/Evaluation/Machine/Cek/Internal.hs#L382

We should indeed put this somewhere more easily accessible and use it.

Reply:
Maybe from `PlutusCore.Core`?

Reply:
@michaelpj, I think I will leave this to a subsequent PR.